src/ua/lviv/iot/algo/part1/lab1/Laptop.java

```diff
@@ -0,0 +1,60 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+
+public class Laptop {
+    private String model = "UNKNOWN";
+    private float screenSize = 15.6f;
+    private int ram = 8;
+    private int storage = 256;
+    private int batteryLife = 5;
+    public int batteryChargeInPercent = 17;
+
+    private static Laptop instance;
+
+    public static Laptop getInstance(){
+        if(instance == null){
+            instance = new Laptop();
+        }
+        return instance;
+    }
+
+    public void addRam(int value){
+        this.ram += value;
+    }
+    public void addStorage(int value){
+        this.storage += value;
+    }
+    public void charge(){
```

Comment:
```suggestion

    public void charge() {
```
Consider applying propper formatting to your codebase